piccolo/apps/playground/commands/run.py

```diff
@@ -252,5 +255,15 @@ def run(
     populate()
 
     from IPython.core.interactiveshell import _asyncio_runner  # type: ignore
+    from traitlets.config import get_config
 
-    IPython.embed(using=_asyncio_runner)
+    if not ipython_profile:
+        conf = get_config()
+        conf.InteractiveShellEmbed.colors = "Neutral"
+    else:
+        print("Using user ipython profile")
+        print("For details, see: https://ipython.org/ipython-doc/3/config/intro.html#ipythondir")
+        print("\n")
+        conf = None
```

Comment:
I would generally flip these, so it's like:

```python
if ipython_profile:
    ...
else:
    ...
```

So the first `if` statement is truthy.